tests/test_tests.py

```diff
@@ -190,3 +190,32 @@ def test_that_a_process_can_be_serialized_and_deserialized():
     process_dict = process.to_dict()
     process_from_dict = Process.from_dict(process_dict)
     assert process_dict == process_from_dict.to_dict()
+
+def test_that_the_grep_date_test_correctly_interprets_exit_code_0_and_1(
```

Comment:
I applaud you for writing the test. I hope we can simplify the testing facilities so it's less work to add new tests.